dist/less/ds6/mixins.less

```diff
@@ -111,3 +111,13 @@
         .background-svg(@ds6-color-link-default);
     }
 }
+
```

Comment:
@RajaRamu Shouldn't the icon mixins be in here? e.g. `icon-close-mixin`? Like we have in ds4? Or maybe I'm misunderstanding how these icons will be used. Can you perhaps give an example of how these changes would be integrated into current ds6 page notice?

Reply:
@ianmcburnie - Based on our discussion, entire `ebay/skin/icon` will be loaded into ebayui-core. Only inline svg `<symbol>` will be aggregated and delivered as inline style dynamically using `marko/transformer`. Usage would be,
1. `<ebay-icon type='background' icon='icon-arrow-down aria-label='arrow down'/>` and output would be    `<span class="icon icon--arrow-down"></span>`.
2. `<ebay-icon type='inline' icon='icon-arrow-down aria-label='arrow down'/>` and output would be `<symbol id="icon-arrow-down" viewBox="0 0 22 22">
        <path d="M7 6.072L4.834 3.906a.656.656 0 0 0-.928.928L6.072 7 3.906 9.166a.656.656 0 0 0 .928.928L7 7.928l2.166 2.166a.656.656 0 0 0 .928-.928L7.928 7l2.166-2.166a.656.656 0 0 0-.928-.928L7 6.072zM7 14A7 7 0 1 1 7 0a7 7 0 0 1 0 14z"/>
    </symbol>` 
&&
 `<svg aria-hidden="true" class="icon icon--arrow-down" focusable="false" height="24" width="24">
                <use xlink:href="#icon-arrow-down"></use>
            </svg>`

`<symbol>` and `<svg>` tags  are in `ds6/icon.html` and `/ds6/svg.html`. i was thinking to extract into some less file so that it could be imported into `ebayui-core`. Contrary to that, `<symbol>` and `<svg>`  could be copied over which is not ideal. thought?

Reply:
@RajaRamu I'm not talking about ebayui-core. I'm talking purely from a skin perspective. How would end-users integrate an icon right now, as I don't see them exposed as mixins in the ds6/mixins. For example, how would the skin page notice integrate the close background icon?

Reply:
@ianmcburnie To use it in `skin` internally, this is what i am thinking. please correct me if i am wrong,
Existing:
```
.page-notice__close {
    border: 0;
    background: none;
    width: 50px;
    height: 48px;
    cursor: pointer;
    span {
        .icon-close;
    }
}
```
will be chagned to,
```
.page-notice__close {
    border: 0;
    background: none;
    width: 50px;
    height: 48px;
    cursor: pointer;
}
```
Existing Template:
```
<section aria-labelledby="information-status" class="page-notice page-notice--information" role="region">
    <h5 class="page-notice__status">
        <span aria-label="Information notice" id="information-status" role="img"></span>
    </h5>
    <span class="page-notice__cell page-notice__cell--align-middle">
        <p>Try refining your search term for more results.</p>
    </span>
    <button aria-label="Dismiss notice" class="page-notice__close" type="button">
        <span aria-hidden="true" role="img"></span>
    </button>
</section>
```
will be changed to 
```
<section aria-labelledby="information-status" class="page-notice page-notice--information" role="region">
    <h5 class="page-notice__status">
        <span aria-label="Information notice" id="information-status" role="img"></span>
    </h5>
    <span class="page-notice__cell page-notice__cell--align-middle">
        <p>Try refining your search term for more results.</p>
    </span>
    <button aria-label="Dismiss notice" class="page-notice__close" type="button">
        <span class='icon icon-close'></span>
    </button>
</section>
```

Reply:
This is incorrect. As I explained in a previous comment, the page notice should not need need to specify `icon icon-close` on the span. Because we know it is always the close icon we can take care of that detail ourselves. Like so:

```less
.page-notice__close span {
  .icon-close-mixin;
}
```

Now no need to specify `icon icon--close`.

```html
<button aria-label="Dismiss notice" class="page-notice__close" type="button">
  <span></span>
</button>
```

Reply:
@RajaRamu Changing the HTML contract doesn't make any sense. The whole idea is to make it _easier_ for developers. This just makes them have to do more.

Additionally, you've removed the necessary accessibility of the icon.

You need to keep things the way they are, and provide mixins as Ian has suggested.

Reply:
@ianmcburnie @seangates - makes sense. i will expose it as a mixin for background svg. how about inline svg. how do deal with <symbol> and <svg>?. should it be exposed as mixin too?.

Reply:
Only if every inline SVG needs some specifically tailored CSS. Like width and height for example.